gui/styles/_wizard.styl

```diff
@@ -81,27 +84,83 @@ a.more-info
   color var(--slateGrey)
 
 
-.folder__selector
-  @extend $form-select select
-  width: 100%
-  font-size: 1.1em
-  line-height: 1em
-  margin 0
-  text-align left
-  text-decoration: none
+.folder__subtitle
+  margin-bottom 0
+
+.folder__config
+  margin-top 1rem
+
+  ul
+    margin-bottom 0
+    padding-inline-start: 1rem;
+
+  li + li
+    margin-top 1rem
+
+.folder__config-option__title
+  font-weight bold
+
+.folder__config-option__button
+  margin-left .5rem
+  color var(--dodgerBlue)
+
+.folder__path
+  position relative
+  padding-left 1.1rem
```

Comment:
Why this value that is not in our padding scale?

Reply:
I'm just making room for the folder icon on the left. Since the icon is `1rem` wide, I add a little extra `0.1rem` to separate it from the text.